### ISSUE TITLE
NodeODM should show tasks that came from WebODM

### ISSUE THREAD

**Issue by gotjoshua** [OpenDroneMap/NodeODM#102]
### What is the problem?

node_odm web interface does not display ongoing tasks that came from web_odm

### What should be the expected behavior?

when web_odm sends a task to a processing node, when i go to the web interface of the processing node it should show the task

### How can we reproduce this? 

1. set up webodm with docker-compose and make the node_odm accessible via traefik (or other reverse proxy). 
2. send a task to node_odm from web_odm (and see that the upload finished and the processing is running):
`webodm_node-odm_2 | info: About to run: /code/run.sh --pc-classify --orthophoto-resolution 4 --dtm --dem-resolution 10 --split-overlap 200 --split 450 --dsm --verbose --project-path /var/www/data 208c37dd-b107-42af-b510-72fbce85834e`
3. go to the front end of node_odm and see that it is ready to accept new tasks, but not showing the task that came from web_odm

**Comment by gotjoshua** [comment]
my docker-compose example:
```yml
version: "2.3"

networks:
  odm-net:
    external: false
  haven:
    external:
      name: haven

volumes:
  dbdata:
    driver: local-persist
    driver_opts:
      mountpoint: /mnt/raid0/srv/volumes/odm-db
  appmedia: #should be == ${WO_MEDIA_DIR}
    driver: local-persist
    driver_opts:
      mountpoint: /mnt/raid0/srv/volumes/odm-appmedia
services:
  db:
    image: opendronemap/webodm_db
    container_name: db
    # ports:
    #   - "5432"
    volumes:
      - dbdata:/var/lib/postgresql/data
    restart: unless-stopped
    oom_score_adj: -100
    networks:
      - odm-net
  webapp:
    image: opendronemap/webodm_webapp
    container_name: webapp
    entrypoint: /bin/bash -c "chmod +x /webodm/*.sh && /bin/bash -c \"/webodm/wait-for-postgres.sh db /webodm/wait-for-it.sh -t 0 broker:6379 -- /webodm/start.sh\""
    volumes:
      - ${WO_MEDIA_DIR}:/webodm/app/media
    expose:
      - 8000
    labels:
      traefik.enable: true
      traefik.app.frontend.rule: Host:odm.<your.tld>
      traefik.app.port: 8000 #internal port to be exposed on https://HOSTNAME
      traefik.docker.network: haven
    depends_on:
      - db
      - broker
      - worker
      - node-odm
    environment:
      - WO_PORT
      - WO_HOST
      - WO_DEBUG
      - WO_BROKER
      - WO_DEV
      - WO_DEFAULT_NODES
    restart: unless-stopped
    oom_score_adj: 0
    networks:
      - odm-net
      - haven
  broker:
    image: redis
    container_name: broker
    restart: unless-stopped
    oom_score_adj: -500
    networks:
      - odm-net
  node-odm:
    image: opendronemap/nodeodm
    container_name: webodm_node-odm_1
    mem_limit: 48g
    memswap_limit: 60g
    cpus: 1.9
    cpuset: 10,11
    volumes:
      - /V1/srv/odm/node1data:/var/www/data
      - /V1/srv/odm/node1tmp:/var/www/tmp
    restart: unless-stopped
    oom_score_adj: 500
    labels:
      traefik.enable: true
      traefik.app.frontend.rule: Host:nodeodm1.<your.tld>
      traefik.app.port: 3000 #internal port to be exposed on https://HOSTNAME
      traefik.docker.network: haven
    networks:
      - odm-net
      - haven
  node-odm_2:
    image: opendronemap/nodeodm
    container_name: webodm_node-odm_2
    mem_limit: 48g
    memswap_limit: 60g
    cpus: 1.9
    cpuset: 8,9
    volumes:
      - /V1/srv/odm/node2data:/var/www/data
      - /V1/srv/odm/node2tmp:/var/www/tmp
    restart: unless-stopped
    oom_score_adj: 500
    labels:
      traefik.enable: true
      traefik.app.frontend.rule: Host:nodeodm2.<your.tld>
      traefik.app.port: 3000 #internal port to be exposed on https://HOSTNAME
      traefik.docker.network: haven
    networks:
      - odm-net
      - haven
  worker:
    image: opendronemap/webodm_webapp
    container_name: worker
    entrypoint: /bin/bash -c "/webodm/wait-for-postgres.sh db /webodm/wait-for-it.sh -t 0 broker:6379 -- /webodm/wait-for-it.sh -t 0 webapp:8000 -- /webodm/worker.sh start"
    volumes:
      - ${WO_MEDIA_DIR}:/webodm/app/media
    depends_on:
      - db
      - broker
    environment:
      - WO_BROKER
      - WO_DEBUG
    restart: unless-stopped
    oom_score_adj: 250
    networks:
      - odm-net
```

**Comment by pierotofy** [comment]
This is now fixed in #99 ! :clinking_glasses:
